Assets/Scenes/Simulator.unity

```diff
@@ -212,9 +212,8 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   _noise: 0
   _reportPeriod: 0.05
-  initGPS:
-  - 0
-  - 0
+  initLat: 0
+  initLon: 0
```

Comment:
What's up with the gps change? Is that vestigial from isaacs stuff?

Reply:
I'm not sure what that is tbh I didn't modify anything else in the scene.